packages/front-end/components/Features/ConditionInput.tsx

```diff
@@ -144,40 +145,50 @@ export default function ConditionInput(props: Props) {
           {conds.map(({ field, operator, value }, i) => {
             const attribute = attributes.get(field);
 
+            if (!attribute) {
```

Comment:
seems like having an attribute for the current field is pretty core to the logic in this file so rather than doing optional chaining on several lines throughout the code I do an early return if we're missing an attribute in our `attributes` Map. Also log an error to the console, but open to other approaches if this doesn't seem like the right move.